sources/packages/web/src/assets/css/base.scss

```diff
@@ -280,6 +280,11 @@ strong {
   color: $button-font-color;
 }
 
+.v-btn.danger-button {
+  background-color: $banner-color-error !important;
+  color: $color-white;
+}
```

Comment:
There is already a red button in the solution and the "red" is achieved using the `v-btn` color defined as `danger`. Is there a reason to not use the same?

![image](https://github.com/user-attachments/assets/88052b5d-52b7-4ed2-b246-65d911c7821f)